AirCasting/Dashboard/DashboardView.swift

```diff
@@ -15,7 +15,7 @@ struct DashboardView: View {
     @StateObject var coreDataHook: CoreDataHook
     @FetchRequest<SensorThreshold>(sortDescriptors: [.init(key: "sensorName", ascending: true)]) var thresholds
     // TODO: - We can rethink the way how "Select Section" works and change it
-    @EnvironmentObject var selectedSection_: SelectSection
+    @EnvironmentObject var selectedSectionNeededForSwipingImplementation: SelectSection
```

Comment:
To be honest I am not sure why you have this property that holds `SelectSection` twice. Maybe you could stay with only this one. If so you could be able to delete a few things below.

About the naming, I personally would leave `Implementation` suffix. 
Only idea, so do it what you want, but maybe: `selectedSectionDesignedForSwiping`?

 If you will stay with only one property then the naming also can change 180 degrees I suppose.